release_docs/INSTALL_CMake.txt

```diff
@@ -872,6 +870,20 @@ The HDF5_ENABLE_COVERAGE option will add "-g -O0 -fprofile-arcs -ftest-coverage"
 to CMAKE_C_FLAGS.
 
 
+========================================================================
+IX. Cross compile HDF5 Libraries with CMake
+========================================================================
+
+HDF5_USE_PREGEN
+HDF5_USE_PREGEN_DIR
+  These CMake variables would allow the use of an appropriate H5Tinit.c file
+  with type information generated on a target platform to be used when cross
+  compiling for those target platforms. Sample pre-generated files are in the
+  folder: config/crosscompile.
```

Comment:
I'm not seeing config/crosscompile.

Can you generate an appropriate H5Tinit.c file b compiling and running h5detect on the target platform, then copying that H5Tinit.c file to $HDF5_USE_PREGEN_DIR?  If so, there should be a sentence or 2 mentioning that.

Reply:
Changed to remove future work.